src/workflow/StreamlitUI.py

```diff
@@ -375,10 +375,16 @@ def input_TOPP(
         # write defaults ini files
         ini_file_path = Path(self.parameter_manager.ini_dir, f"{topp_tool_name}.ini")
         if not ini_file_path.exists():
+            
+            # in local mode the TOPP tools in bin
             if st.session_state.location == "local":
```

Comment:
yeah agree with you.
how should fixed then I think the TOPP Tools should placed in same directory so will work for both. 
or should create other mode (this will more prefer for me)?

Reply:
because there are many things to change while executable, if we add other mode.